src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/contactperson/ContactPersonService.kt

```diff
@@ -276,10 +279,40 @@ class ContactPersonService(
         }
     }.let { PrisonerWithRestrictions(restrictions = it) }
 
+  fun findPersonIdsByFilter(
```

Comment:
We could drop these filters and simplify since it is just used for reconciliation